src/runtime/threading_backend.cc

```diff
@@ -133,22 +133,39 @@ class ThreadGroup::Impl {
           sizeof(cpu_set_t), &cpuset);
 #endif
     }
-    if (exclude_worker0) {  // bind the master thread to core 0
-      cpu_set_t cpuset;
-      CPU_ZERO(&cpuset);
-      if (reverse) {
-        CPU_SET(sorted_order_[sorted_order_.size() - 1], &cpuset);
-      } else {
-        CPU_SET(sorted_order_[0], &cpuset);
+    if (exclude_worker0) {  // master thread run task
+#if defined(__ANDROID__)
+      SetFullCpuAffinity();
```

Comment:
At android, let us still bind the master thread(unless there is a strong reason not to do so), but avoid the pthread_atfork callback(because it is not available)

Reply:
If at android, we still bind master thread like before, I worry about OpenCV + TVM still meet the same issue on linux described here

Reply:
I agree that we don't bind master thread at Android.

Reply:
OK, i am OK with the decision given @yidawang also agrees